mantle/cmd/kola/options.go

```diff
@@ -325,6 +326,17 @@ func syncCosaOptions() error {
 				}
 			}
 		}
+	case "gce":
+		// Pick up the GCP image from the build metadata unless the
+		// user passed in something other than the default
+		if kola.GCEOptions.Image == defaultGCEImage {
```

Comment:
Hmm, wouldn't it be safer to have the default be `""` and then here auto-generate it if there's a cosa build and otherwise fallback to `defaultGCEImage`?

Reply:
>  and otherwise fallback to `defaultGCEImage`?

The problem is. I don't see a good place to do that part ^^ - I guess we could do it under the `if foundCosa && useCosa {}` block but that seemed equally fragile if someone moves that code around.

The difference here between GCP and AWS is that for AWS there is no sane default so we must use `""` as the default. With GCP the sane default is the pointer image to the latest in the image family (like a moving tag).

Reply:
though.. we don't have to set a default for GCP and we could just set it to `""` as well and get rid of `defaultGCEImage`.

Reply:
Would it make sense to just not have a default at all now that we have auto-detection from the cosa build? This matches what we do for AWS.

Reply:
Bahh sorry, hadn't seen your second comment. :)

Reply:
updated